presets-create-react-app doesn't work on windows.

With version 1.2.0, on Windows I get the following error when I start Storybook:

```
ERR! Error: Cannot find module 'C:\Dev\10_poc\storybook\node_modules\react-scripts\bin\react-scripts.js"\config\webpack.config'
ERR! Require stack:
ERR! - C:\Dev\10_poc\storybook\node_modules\@storybook\preset-create-react-app\index.js
```

I guess you need to strip `\bin\react-scripts.js` from the path.

Thank you

--- a/packages/preset-create-react-app/helpers/getReactScriptsPath.js
+++ b/packages/preset-create-react-app/helpers/getReactScriptsPath.js
@@ -13,7 +13,7 @@ const getReactScriptsPath = () => {
     try {
       const content = fs.readFileSync(scriptsBinPath, 'utf8');
       const packagePathMatch = content.match(
-        /"\$basedir[\\/]([^\s]+?[\\/]bin[\\/]react-scripts\.js")/i
+        /"\$basedir[\\/](\S+?)[\\/]bin[\\/]react-scripts\.js"/i
       );
 
       if (packagePathMatch && packagePathMatch.length > 1) {

